apps/yarn.lock

```diff
@@ -7659,7 +7659,7 @@ __metadata:
     "@code-dot-org/artist": 0.2.1
     "@code-dot-org/blockly": 4.0.14
     "@code-dot-org/craft": 0.2.2
-    "@code-dot-org/dance-party": 1.0.12
+    "@code-dot-org/dance-party": 1.0.13
```

Comment:
@sanchitmalhotra126 - FYI, added this upgrade to capture changes from https://github.com/code-dot-org/dance-party/pull/664